primitives/linux-port/column.cpp

```diff
@@ -1420,6 +1572,38 @@ void vectorizedFiltering(NewColRequestHeader* in, ColResultHeader* out, const T*
     }
   }
 
+  MT* nonEmptyMaskAux;
+
+  if constexpr (IS_AUX_COLUMN)
```

Comment:
Did I get it right that if IS_AUX_COLUMN is true we process aux column buffer prior to the actual column buffer?

Reply:
I can, but what is the point of creating a separate template function here?

Reply:
I have created a separate function for this if block.